### DISCUSSION TITLE
Is smarter dimensioning possible?

### DISCUSSION THREAD

**Discussion by pkrpkrusr** [jgraph/drawio#3045] [Ideas]
Just wondering if there's a way to show dimensions next to a shape eg the length of a line. 
It's possible to draw a dimension line eg. a line with arrows at the end, and then enter a dimension manually as a text label.
However if you change the line length, the dimension is different and the label has to be adjusted manually to reflect the new value.
Is there a way to do this smarter, perhaps via properties that can be shown as label text that shows the new value automatically? 
I'm a newbie with this app, just an open thought for something that perhaps exists already?
It would be something valuable for engineering and floorplan drawings.

**Comment by zoda61** [comment]
Hello, I just wish the same, special to make floorplan drawings. Not engineering exactly but easier creation for true size layouts. Connecting endpoints of the walls, windows, doors for dinamic changes of the layout measurement without to loose the connections.
